examples/payment-channel/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "payment_channel"
+version = "3.0.1"
```

Comment:
Can you merge master and bump this to `4.0`?

Reply:
I had already merged master. Bumped the version